config/development_polygon.toml

```diff
@@ -0,0 +1,216 @@
+[app]
+chain_id = 137
+
+# a database is optional. it is used for user authentication and accounting
+# TODO: how do we find the optimal db_max_connections? too high actually ends up being slower
+db_max_connections = 20
+# development runs cargo commands on the host and so uses "mysql://root:dev_web3_proxy@127.0.0.1:13306/dev_web3_proxy" for db_url
+# production runs inside docker and so uses "mysql://root:web3_proxy@db:3306/web3_proxy" for db_url
+db_url = "mysql://root:dev_web3_proxy@127.0.0.1:13306/dev_web3_proxy"
+
+# deposit_contract = "0xc9fcfa7e28ff320c49967f4522ebc709aa1fde7c"
+deposit_contract = "0x4e3bc2054788de923a04936c6addb99a05b0ea36"
```

Comment:
this has the contract, but no topic. is the topic still needed?

Reply:
good catch, needed indeed, added it